modules/aws-kernel/src-jvm/smithy4s/aws/SysEnvPlatformCompat.scala

```diff
@@ -17,5 +17,5 @@
 package smithy4s.aws.kernel
 
 trait SysEnvPlatformCompat {
-  def readEnvVar(envVar: String): Option[String] = sys.env.get(envVar)
+  def readEnvVar(envVar: String): Option[String] = Option(System.getenv(envVar))
```

Comment:
This isn't strictly necessary, but I think when SecurityManager is enabled, reading an individual variable instead of the whole environment might be better for permission scope.